paasta_tools/cli/utils.py

```diff
@@ -493,10 +494,10 @@ def run_paasta_serviceinit(subcommand, master, service, instances, cluster, stre
     else:
         delta_flag = ''
 
-    ssh_flag = '-t' if stream else '-n'
+    ssh_flags += '-t' if stream else '-n'
```

Comment:
I see that you are explicitly adding trailing whitespace when you pass in some `ssh_flags`. That seems like a very easy thing for someone to overlook in the future. What if we added the whitespace here instead?